colearn/utils/results.py

```diff
@@ -25,6 +25,7 @@ def __init__(self):
         self.votes = []
         self.test_scores = []
         self.vote_scores = []
+        self.privacy_budgets = []
```

Comment:
I remember a discussion from the past, that we do not want to show DP at too many places in the codebase, and restrict it only as on optional feature.

Wouldn't it makes more sense to omit the privacy_budget from the results.py? Or maybe extracting the needed info throught the TrainingSummary instead?

Reply:
yes, make sens

Reply:
it is actually TrainingSummary that is used. Renamed it to training_summaries